specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14/examples/createVault.json

```diff
@@ -0,0 +1,204 @@
+{
+    "parameters":{
+       "resourceGroupName":"sample-resource-group",
+       "vaultName":"sample-vault",
+       "subscriptionId":"00000000-0000-0000-0000-000000000000",
+       "api-version":"2018-02-14-preiview",
+       "parameters":{
+            "location": "westus", 
+            "properties":{
+                "tenantId": "00000000-0000-0000-0000-000000000000",
+                "sku": {"family": "A", "name": "standard"},
+                "accessPolicies":[{
+                    "tenantId": "00000000-0000-0000-0000-000000000000",
+                    "objectId": "00000000-0000-0000-0000-000000000000",
+                    "permissions": {
+                        "keys": [
+                            "encrypt",
+                            "decrypt",
+                            "wrapKey",
+                            "unwrapKey", 
+                            "sign",
+                            "verify",
+                            "get",
+                            "list",
+                            "create",
+                            "update",
+                            "import",
+                            "delete",
+                            "backup",
+                            "restore",
+                            "recover",
+                            "purge"
```

Comment:
I would not include the 'purge' permission in examples. It should be expressly and judiciously assigned, and samples have a tendency of being copied and pasted.